src/main/kotlin/edu/wpi/first/wpilib/javastyle/WPILibJavaStylePluginExtension.kt

```diff
@@ -1,10 +1,11 @@
 package edu.wpi.first.wpilib.javastyle
 
 open class WPILibJavaStylePluginExtension {
```

Comment:
Not saying you need to use it but just letting you know that this exists: 
https://github.com/gradle/kotlin-dsl/releases/tag/v0.9.0

You can do this sort of thing:
```kotlin
open class GreetingPluginExtension(project: Project) {

    // Declare a `PropertyState<String>` backing field
    private
    val messageState = project.property<String>()

    // Expose `messageState` as the `message` property whose type is inferred as String
    var message by messageState

    // Can also be exposed as `Provider<String>` for additional functionality
    val messageProvider: Provider<String> get() = messageState

    // `outputFiles` property type is inferred as `ConfigurableFileCollection`
    // with the following behaviour:
    //  - getting will always return the original instance
    //  - setting will `setFrom` the source
    var outputFiles by project.files()
}
```

Reply:
I'll look into this later.